dashboard/app/controllers/sessions_controller.rb

```diff
@@ -8,7 +8,7 @@ class SessionsController < Devise::SessionsController
   def new
     session[:user_return_to] ||= params[:user_return_to]
     if params[:maker]
-      redirect_to user_google_oauth2_omniauth_authorize_path
+      redirect_to "/maker/google_oauth_confirm_login"
```

Comment:
nit: use the path helper here

there's a formula for how to figure out the helper on your own, but i never remember it (somewhere in [Rails Routing from the Outside In](https://guides.rubyonrails.org/routing.html#prefixing-the-named-route-helpers), i think). instead, if you go to any non-existent route on localhost (e.g., http://localhost-studio.code.org:3000/rails), it'll show you an error page that tells you all the routes + their helpers that do exist!